### PR TITLE
Jquery once fix

### PR DESCRIPTION
**Pull request by wgilling**
**GitHub Issue**: (link)
https://github.com/Islandora/documentation/issues/2205

# What does this Pull Request do?
Minor adjustment to the openseadragon javascript which connects the viewer to the underlying image resource

# intended result of the PR will be and/or what problem it solves.
Prevents additional canvas elements from being shown in and below an OpenSeadragon viewer.

# Technical details and possible side effects.
unknown

# How should this be tested?
BEFORE this code is merged, it is important to find an islandora object with image media that will use the OpenSeadragon viewer -- which causes the doubled-canvas output. (See https://github.com/Islandora/documentation/issues/2205). I have found that this happens when a larger resolution service file JPG is uploaded for a Service File media.
After the code has been merged, that same object from above should only display one canvas.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers

### CODE REVIEW

File: js/openseadragon_viewer.js [hunk 1]
```diff
@@ -21,7 +21,7 @@
             Object.keys(settings.openseadragon).forEach(function(osdViewerId) {
               // Use custom element #id if set.
               base = '#' + osdViewerId;
-              $(base, context).once('openSeadragonViewer').each(function () {
+              once('openSeadragonViewer', $(base, context)).forEach(function () {
```
Comment (adam-vessey) [inline js/openseadragon_viewer.js]: Still needs `jQuery` for the `$(base, context)` thing, no?... unless `once()` itself supports similar syntax?

Comment (aOelschlager) [reply js/openseadragon_viewer.js]: You're right, I need to revert back to how I had this in my second commit.

Comment (adam-vessey) [reply js/openseadragon_viewer.js]: Hmm... not sure that jQuery is actually necessary, checking out the change log for `once()`. Should be able to have something like:

```js
              once('openSeadragonViewer', base, context).forEach(function () {
```

Comment (adam-vessey) [reply js/openseadragon_viewer.js]: https://github.com/aOelschlager/openseadragon/pull/1 might do the trick?

**Review by seth-shaw-asu — APPROVED** [review]
:+1:

**Pull request merged**